workflow/metrics/server.go

```diff
@@ -150,3 +165,9 @@ func (m *Metrics) garbageCollector(ctx context.Context) {
 		}
 	}
 }
+
+// recordBuildInfo publishes information about Argo-Rollouts version and runtime info through an info metric (gauge).
```

Comment:
this isn't Argo Rollouts
```suggestion
// recordBuildInfo publishes metadata about Argo's version and runtime through a metric (gauge).
```